SmartDeviceLink/SDLSyncMsgVersion.h

```diff
@@ -30,6 +30,12 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (strong, nonatomic) NSNumber<SDLInt> *minorVersion;
 
+/**
+ * @abstract Optional, allows backward-compatible fixes to the API without increasing the minor version of the interface
+ *
+ */
+@property (strong, nonatomic) NSNumber<SDLInt> *patchVersion;
```

Comment:
This should be nullable as the MOBILE_API XML spec specifies that this is not a mandatory parameter and may not exist.